src/test/java/net/sf/jabref/exporter/MsBibExportFormatTest.java

```diff
@@ -0,0 +1,58 @@
+package net.sf.jabref.exporter;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+import java.util.Collections;
+import java.util.List;
+
+import net.sf.jabref.BibDatabaseContext;
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.MetaData;
+import net.sf.jabref.model.database.BibDatabase;
+import net.sf.jabref.model.entry.BibEntry;
+
+import com.google.common.base.Charsets;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import static org.junit.Assert.assertEquals;
+
+public class MsBibExportFormatTest {
+
+    public static BibDatabaseContext databaseContext;
+    public static Charset charset;
+    public List<BibEntry> entries;
```

Comment:
Inline the entries field since it is not used across tests or initialized in the setup method.